src/pysdmx/io/xml/sdmx_two_one/writer/__write_aux.py

```diff
@@ -0,0 +1,230 @@
+"""Writer auxiliary functions."""
+
+from collections import OrderedDict
+from typing import Any, Dict
+
+from pysdmx.model.message import Header, MessageType
+
+MESSAGE_TYPE_MAPPING = {
+    MessageType.GenericDataSet: "GenericData",
+    MessageType.StructureSpecificDataSet: "StructureSpecificData",
+    MessageType.Metadata: "Structure",
```

Comment:
This could be renamed to MessageType.Structure. I mention this because there are also reference metadata messages, which is what people might be thinking about when seeing MessageType.Metadata.

Reply:
Changing the MessageType to Structure.